src/core/Akka/Event/ILoggingAdapter.cs

```diff
@@ -6,246 +6,748 @@
-        public bool IsWarningEnabled { get { return false; } }
+        bool IsWarningEnabled { get; }
 
         /// <summary>
-        /// Check to determine whether the <see cref="LogLevel.ErrorLevel" /> is enabled.
+        /// Check to determine whether the <see cref="LogLevel.ErrorLevel"/> is enabled.
         /// </summary>
-        public bool IsErrorEnabled { get { return false; } }
+        bool IsErrorEnabled { get; }
 
         /// <summary>
         /// Determines whether a specific log level is enabled.
         /// </summary>
         /// <param name="logLevel">The log level that is being checked.</param>
-        /// <returns>
-        ///   <c>true</c> if the specified level is enabled; otherwise <c>false</c>.
-        /// </returns>
-        public bool IsEnabled(LogLevel logLevel)
-        {
-            return false;
-        }
+        /// <returns><c>true</c> if the specified level is enabled; otherwise <c>false</c>.</returns>
+        bool IsEnabled(LogLevel logLevel);
 
-        /// <summary>
-        /// Logs a <see cref="LogLevel.DebugLevel" /> message.
-        /// </summary>
-        /// <param name="format">The message that is being logged.</param>
-        /// <param name="args">An optional list of items used to format the message.</param>
-        public void Debug(string format, params object[] args) { }
+        // /// <summary>
+        // /// Logs a message with a specified level.
+        // /// </summary>
+        // /// <param name="logLevel">The level used to log the message.</param>
+        // /// <param name="format">The message that is being logged.</param>
+        // /// <param name="args">An optional list of items used to format the message.</param>
+        // void Log(LogLevel logLevel, string format, params object[] args);
 
         /// <summary>
-        /// Logs a <see cref="LogLevel.ErrorLevel" /> message and associated exception.
+        /// Logs a message with a specified level.
         /// </summary>
-        /// <param name="cause">The exception associated with this message.</param>
+        /// <param name="logLevel">The level used to log the message.</param>
+        /// <param name="cause">The exception that caused this log message.</param>
         /// <param name="format">The message that is being logged.</param>
         /// <param name="args">An optional list of items used to format the message.</param>
-        public void Debug(Exception cause, string format, params object[] args){ }
+        void Log(LogLevel logLevel, Exception cause, string format);
 
-        /// <summary>
-        /// Logs a <see cref="LogLevel.InfoLevel" /> message.
-        /// </summary>
-        /// <param name="format">The message that is being logged.</param>
-        /// <param name="args">An optional list of items used to format the message.</param>
-        public void Info(string format, params object[] args) { }
+        void Log(LogLevel logLevel, Exception cause, LogMessage message);
```

Comment:
`ILoggingAdapter` only supports two direct logging calls now:

```csharp
void Log(LogLevel logLevel, Exception cause, LogMessage message);
void Log(LogLevel logLevel, Exception cause, string format);
```

Everything else is inputted via the extension methods earlier in the file.